docs/source/nodes_and_pipelines/nodes.md

```diff
@@ -192,6 +192,6 @@ The output is as follows:
 Out[2]: {'sum': 5}
 ```
 
-```eval_rst
-.. note::  It is also possible to call a node as a regular Python function: ``adder_node(dict(a=2, b=3))``. This will call ``adder_node.run(dict(a=2, b=3))`` behind the scenes.
+```{note}
+It is also possible to call a node as a regular Python function: ``adder_node(dict(a=2, b=3))``. This will call ``adder_node.run(dict(a=2, b=3))`` behind the scenes.
```

Comment:
```suggestion
It is also possible to call a node as a regular Python function: `adder_node(dict(a=2, b=3))`. This will call `adder_node.run(dict(a=2, b=3))` behind the scenes.
```